bin/cmap2bed.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python 
+
+import optparse
+import sys
+import io
+import string
+
+<<<<<<< HEAD
+parser = optparse.OptionParser()
+=======
+# Script originally developed by Yumi Sims (yy5@sanger.ac.uk)
+
+parser = optparse.OptionParser(version="%prog 1.0")
+>>>>>>> f3007388302cde3cecfe7a50820849432b0c8b12
```

Comment:
There's a merge conflict here. Can you also make sure the script supports `--version` ?